### PR TITLE
Fix: #175 Documentation for the bank package.

### LINKED ISSUE #175: Document the files in mifopay/bank package
**Summary:**

Document the files in mifopay/bank package: https://github.com/openMF/mobile-wallet/tree/master/mifospay/src/main/java/org/mifos/mobilewallet/mifospay/bank

### PR DESCRIPTION
**Pull request by nikhil-sachdeva**
Fixes #175

Documentation of the bank package.

### CODE REVIEW

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/bank/adapters/OtherBankAdapter.java [hunk 1]
```diff
@@ -74,6 +75,11 @@ public Bank getBank(int position) {
         return otherBanks.get(position);
     }
 
+    /**
+     * Method to apply filter on the bank account list based on the text entered in search bar.
+     * @return filter.
+     */
+
```
Comment (luckyman20) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/bank/adapters/OtherBankAdapter.java]:
```suggestion
```

Comment (nikhil-sachdeva) [reply mifospay/src/main/java/org/mifos/mobilewallet/mifospay/bank/adapters/OtherBankAdapter.java]: Ok I'll look into it.

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/bank/ui/LinkBankAccountActivity.java [hunk 2]
```diff
@@ -76,6 +76,10 @@ protected void onCreate(Bundle savedInstanceState) {
         hideProgressDialog();
     }
 
+    /**
+     * Method to setup RecyclerViews with adapters.
+     */
+
```
Comment (luckyman20) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/bank/ui/LinkBankAccountActivity.java]:
```suggestion
```

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/bank/ui/LinkBankAccountActivity.java [hunk 3]
```diff
@@ -136,6 +140,11 @@ public void afterTextChanged(Editable s) {
         });
     }
 
+    /**
+     * Method to filter the list based on the text entered.
+     * @param text
+     */
+
```
Comment (luckyman20) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/bank/ui/LinkBankAccountActivity.java]:
```suggestion
```

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/bank/BankContract.java [hunk 4]
```diff
@@ -7,9 +7,13 @@
 import java.util.List;
 
 /**
- * Created by ankur on 09/July/2018
+ * This is a contract working as an interface between
+ * UI and Presenter of the bank section.
```
Comment (luckyman20) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/bank/BankContract.java]: Keep it in a single line.

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/bank/ui/BankAccountDetailActivity.java [hunk 5]
```diff
@@ -26,6 +26,10 @@
 import butterknife.ButterKnife;
 import butterknife.OnClick;
 
+/**
+ * This the activity for bank account details.
+ */
+
```
Comment (luckyman20) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/bank/ui/BankAccountDetailActivity.java]:
```suggestion
```

**Comment by jsahil730** [comment]
@nikhil-sachdeva please do resolve the conflicts, if you want your PR to be accepted, else please close this PR.

**Pull request closed without merging**